react/features/chat/functions.ts

```diff
@@ -190,3 +191,17 @@ export function getPrivateNoticeMessage(message: IMessage) {
         recipient: message.messageType === MESSAGE_TYPE_LOCAL ? message.recipient : i18next.t('chat.you')
     });
 }
+
+
+/**
+ * Check if participant is not allowed to send group messages.
+ *
+ * @param {IReduxState} state - The redux state.
+ * @returns {boolean} - Returns true if the participant is not allowed to send group messages.
+ */
+export function isSendGroupChatDisabled(state: IReduxState) {
+    const isJwtGroupChatEnabled = isJwtFeatureEnabled(state, 'send-groupchat', false, false);
```

Comment:
If token is missing group chat is disabled? If the feature is missing also group chat is disabled? 
That should be the other way around. We should only be able to explicitly disable this.